commcare_connect/opportunity/forms.py

```diff
@@ -103,35 +103,54 @@ class Meta:
         self.helper.layout = Layout(
-            Row(Field("name")),
-            Row(Field("active", css_class="form-check-input", wrapper_class="form-check form-switch")),
-            Row(Field("is_test", css_class="form-check-input", wrapper_class="form-check form-switch")),
-            Row(Field("delivery_type")),
-            Row(Field("description")),
-            Row(Field("short_description")),
             Row(
-                Field("currency", wrapper_class="form-group col-md-6 mb-0"),
-                Field("end_date", wrapper_class="form-group col-md-6 mb-0"),
+                HTML("<div class='col-span-2'><h6 class='title-sm'>Opportunity Details</h6>  <span class='hint'>Edit the details of the opportunity. All fields are mandatory.  </span> </div>"),
+                Column(
+                    Field("name", wrapper_class="w-full"),
+                    Field("short_description", wrapper_class="w-full"),
+                    Field("description", wrapper_class="w-full"),
+                ),
+                Column(
+                    Field("delivery_type"),
+                    Field("active", css_class=CHECKBOX_CLASS, wrapper_class="bg-slate-100 flex items-center justify-between p-4 rounded-lg"),
+                    Field("is_test", css_class=CHECKBOX_CLASS, wrapper_class="bg-slate-100 flex items-center justify-between p-4 rounded-lg"),
+                ),
+                css_class="grid grid-cols-2 gap-4 p-6 card_bg"
             ),
-            HTML("<hr />"),
-            Fieldset(
-                "Invite Users",
-                Row(Field("users")),
+            Row(
+                HTML("<div class='col-span-2'><h6 class='title-sm'>Date</h6>  <span class='hint'>Optional: If not specified, the opportunity start & end dates will apply to the form submissions.</span> </div>"),
+                Column(
+                    Field("end_date"),
+                ),
+                Column(
+                    Field("currency"),
+                    Field("additional_users")
+                ),
+                css_class="grid grid-cols-2 gap-4 p-6 card_bg"
+            ),
+            Row(
+                HTML("<div class='col-span-2'><h6 class='title-sm'>Invite Workers</h6></div>"),
                 Row(
-                    Field("filter_country", wrapper_class="form-group col-md-6 mb-0"),
-                    Field("filter_credential", wrapper_class="form-group col-md-6 mb-0"),
+                    Field("filter_country", wrapper_class="w-full"),
+                    Field("filter_credential", wrapper_class="w-full"),
+                    css_class="flex gap-2"
                 ),
+                Row(Field("users", wrapper_class="w-full"), css_class="col-span-2"),
+                css_class="grid grid-cols-2 gap-4 p-6 card_bg"
             ),
-            Submit("submit", "Submit"),
+            Row(
+                Submit("submit", "Submit", css_class="button button-md primary-dark"),
```

Comment:
Adding float-end to the button style will have the same effect as flex justify-end, also you can remove the row here since it contains a single element and it will be wrapped in a div by default.

Reply:
I don't think it gets wrapped in Div without Row. I have used float-end though, that's good idea.